flo-runner/src/test/java/com/spotify/flo/context/FloRunnerTest.java

```diff
@@ -439,7 +440,7 @@ public void mockingInputsOutputsAndContextShouldBePossibleInTestScope() throws E
             return publisher.publish(result);
           });
 
-      assertThat(runTask(task).future().get(30, SECONDS), is(jobResult));
+      assertThat(runTask(task).future().get(30, DAYS), is(jobResult));
```

Comment:
A bit too patient maybe? Would this hang the build if things go wrong?

Reply:
woops, debugging leftover =)